learntools/ml-insights/nbs/ch2.ipynb

```diff
@@ -0,0 +1,889 @@
+       "      <td>40.769592</td>\n",
+       "      <td>-73.964015</td>\n",
+       "      <td>40.770022</td>\n",
+       "      <td>2.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>max</th>\n",
+       "      <td>-73.800652</td>\n",
+       "      <td>40.799952</td>\n",
+       "      <td>-73.800102</td>\n",
+       "      <td>40.799999</td>\n",
+       "      <td>6.000000</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       pickup_longitude  pickup_latitude  dropoff_longitude  dropoff_latitude  \\\n",
+       "count      25775.000000     25775.000000       25775.000000      25775.000000   \n",
+       "mean         -73.973905        40.755495         -73.971926         40.754594   \n",
+       "std            0.023460         0.020579           0.024684          0.023701   \n",
+       "min          -73.999999        40.604037         -73.999999         40.602453   \n",
+       "25%          -73.988016        40.743961         -73.986749         40.743833   \n",
+       "50%          -73.979410        40.757694         -73.977976         40.757740   \n",
+       "75%          -73.966730        40.769592         -73.964015         40.770022   \n",
+       "max          -73.800652        40.799952         -73.800102         40.799999   \n",
+       "\n",
+       "       passenger_count  \n",
+       "count     25775.000000  \n",
+       "mean          1.654045  \n",
+       "std           1.281060  \n",
+       "min           0.000000  \n",
+       "25%           1.000000  \n",
+       "50%           1.000000  \n",
+       "75%           2.000000  \n",
+       "max           6.000000  "
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_X.describe()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "count    23466.000000\n",
+       "mean         8.472539\n",
+       "std          4.609747\n",
+       "min          0.010000\n",
+       "25%          5.500000\n",
+       "50%          7.500000\n",
+       "75%         10.100000\n",
+       "max        165.000000\n",
+       "Name: fare_amount, dtype: float64"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_y.describe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1\n",
+    "\n",
+    "The first model uses the following featuers\n",
+    "- pickup_longitude\n",
+    "- pickup_latitude\n",
+    "- dropooff_longitude\n",
+    "- dropoff_latitude\n",
+    "- passenger_count\n",
+    "\n",
+    "Before running any code... what do you expect the importances from this model might look like?\n",
```

Comment:
Since PI is a property of a model, not of the problem/data, it might be worth mentioning at this point what sort of model this is. Latitude and longitude seem like very useful features, but if we were working with a linear regression model, I'd expect them to have pretty low PI.